Description: Fix GPU on Windows Subsystem for Linux
Origin: https://github.com/OGRECave/ogre-next/pull/388

diff --git a/RenderSystems/GL3Plus/src/OgreGL3PlusTextureGpu.cpp b/RenderSystems/GL3Plus/src/OgreGL3PlusTextureGpu.cpp
index c9a1147b188..85a94ae7a71 100644
--- a/RenderSystems/GL3Plus/src/OgreGL3PlusTextureGpu.cpp
+++ b/RenderSystems/GL3Plus/src/OgreGL3PlusTextureGpu.cpp
@@ -649,15 +649,23 @@ namespace Ogre
                                           dstBox.getZOrSlice() + dstGl->getInternalSliceStart(),
                                           srcBox.width, srcBox.height, srcBox.getDepthOrSlices() ) );
             }
-            /*TODO
             else if( support.checkExtension( "GL_NV_copy_image" ) )
-            {
-                OCGE( glCopyImageSubDataNV( this->mFinalTextureName, this->mGlTextureTarget,
-                                            srcMipLevel, srcBox.x, srcBox.y, srcBox.z,
-                                            dstGl->mFinalTextureName, dstGl->mGlTextureTarget,
-                                            dstMipLevel, dstBox.x, dstBox.y, dstBox.z,
-                                            srcBox.width, srcBox.height, srcBox.getDepthOrSlices() ) );
-            }*/
+            {
+                 // Initialize the pointer only the first time
+                PFNGLCOPYIMAGESUBDATANVPROC local_glCopyImageSubDataNV = nullptr;
+                if (!local_glCopyImageSubDataNV)
+                {
+                    local_glCopyImageSubDataNV = (PFNGLCOPYIMAGESUBDATANVPROC)gl3wGetProcAddress("glCopyImageSubDataNV");
+                }
+
+                OCGE( local_glCopyImageSubDataNV( this->mFinalTextureName, this->mGlTextureTarget,
+                                                  srcMipLevel, srcBox.x, srcBox.y,
+                                                  srcBox.getZOrSlice() + this->getInternalSliceStart(),
+                                                  dstGl->mFinalTextureName, dstGl->mGlTextureTarget,
+                                                  dstMipLevel, dstBox.x, dstBox.y,
+                                                  dstBox.getZOrSlice() + dstGl->getInternalSliceStart(),
+                                                  srcBox.width, srcBox.height, srcBox.getDepthOrSlices() ) );
+            }
             /*TODO: These are for OpenGL ES 3.0+
             else if( support.checkExtension( "GL_OES_copy_image" ) )
             {
