diff --git a/CMakeLists.txt b/CMakeLists.txt
index ac5f96c..df51da6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,8 +10,8 @@ include(ExternalProject)
 ExternalProject_Add(astra_openni2
   PREFIX astra_openni2
   # SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/astra_openni2/OpenNI2
-  GIT_REPOSITORY https://github.com/tfoote/OpenNI2.git
-  GIT_TAG orbbec_ros
+  GIT_REPOSITORY https://github.com/occipital/OpenNI2.git
+  GIT_TAG master
   CONFIGURE_COMMAND echo "no need to configure"
   #${CMAKE_CURRENT_SOURCE_DIR}/libantlr/configure --prefix=<INSTALL_DIR>
   BUILD_IN_SOURCE 1
