From 4ec85f4eaaea8525e031e6734620ae290badd07c Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Wed, 17 Jan 2018 01:12:00 +0000
Subject: [PATCH] s/logWarn/CONSOLE_BRIDGE_logWarn

---
 test/cleanup.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/cleanup.cpp b/test/cleanup.cpp
index efef490..ede9a44 100644
--- a/test/cleanup.cpp
+++ b/test/cleanup.cpp
@@ -3,10 +3,10 @@
 
 struct A {
   A(const char* hint) {
-    logWarn("initializing class: %s", hint);
+    CONSOLE_BRIDGE_logWarn("initializing class: %s", hint);
   }
   ~A() {
-    logWarn("destroying class");
+    CONSOLE_BRIDGE_logWarn("destroying class");
   }
 };
 
-- 
2.15.1
