From 198f092fdcb9045e4d50fcdb5cf15ab8042dafad Mon Sep 17 00:00:00 2001
From: Sammy Pfeiffer <Sammy.Pfeiffer@student.uts.edu.au>
Date: Wed, 16 Jan 2019 00:16:37 +1100
Subject: [PATCH] Prefer Gstreamer 1.0 over 0.10

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 98166b7..7d34230 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,8 +5,8 @@ project(gscam)
 # System Dependencies
 find_package(PkgConfig)
 
-pkg_check_modules(GSTREAMER QUIET gstreamer-0.10)
-if(NOT GSTREAMER_FOUND)
+pkg_check_modules(GSTREAMER QUIET gstreamer-1.0)
+if(GSTREAMER_FOUND)
   set(GSTREAMER_VERSION_1_x TRUE)
 endif()
 if(GSTREAMER_VERSION_1_x)
-- 
2.7.4

