diff --git a/cfg/CompressedPublisher.cfg b/cfg/CompressedPublisher.cfg
index 856ba59..418680d 100755
--- a/cfg/CompressedPublisher.cfg
+++ b/cfg/CompressedPublisher.cfg
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/ros-python-kinetic
 
 PACKAGE='compressed_image_transport'
 
diff --git a/cfg/CompressedSubscriber.cfg b/cfg/CompressedSubscriber.cfg
index 454f630..4bf2c65 100755
--- a/cfg/CompressedSubscriber.cfg
+++ b/cfg/CompressedSubscriber.cfg
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/ros-python-kinetic
 
 PACKAGE='compressed_image_transport'
 
