From f7dd0046920f2950a15e1d00fd6e91db783949b7 Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Sat, 7 Jul 2018 16:01:44 -0400
Subject: [PATCH 3/4] Force non-debian layout.

---
 cmake/python.cmake | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/cmake/python.cmake b/cmake/python.cmake
index c72b859..cfc9531 100644
--- a/cmake/python.cmake
+++ b/cmake/python.cmake
@@ -8,9 +8,10 @@ message(STATUS "Using PYTHON_EXECUTABLE: ${PYTHON_EXECUTABLE}")
 set(_PYTHON_PATH_VERSION_SUFFIX "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
 
 set(enable_setuptools_deb_layout OFF)
-if(EXISTS "/etc/debian_version")
-  set(enable_setuptools_deb_layout ON)
-endif()
+# We know we are in Gentoo, it should be OFF (Gentoo Prefix fails to build otherwise)
+# if(EXISTS "/etc/debian_version")
+#   set(enable_setuptools_deb_layout ON)
+# endif()
 option(SETUPTOOLS_DEB_LAYOUT "Enable debian style python package layout" ${enable_setuptools_deb_layout})
 
 if(SETUPTOOLS_DEB_LAYOUT)
-- 
2.18.0

