diff --git a/CMakeLists.txt b/CMakeLists.txt
index fc5384d..48904cb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -58,7 +58,7 @@ if (DOC_ONLY)
   return()
 endif()
 
-enable_testing()
+# enable_testing()
 
 # Use GNUInstallDirst to get canonical paths
 include(GNUInstallDirs)
@@ -120,9 +120,9 @@ endif()
 
 ######################################
 # Enable screen tests by default
-if(NOT DEFINED ENABLE_SCREEN_TESTS)
-  set(ENABLE_SCREEN_TESTS TRUE)
-endif()
+# if(NOT DEFINED ENABLE_SCREEN_TESTS)
+#   set(ENABLE_SCREEN_TESTS TRUE)
+# endif()
 
 message (STATUS "\n\n====== Finding 3rd Party Packages ======")
 include (${gazebo_cmake_dir}/SearchForStuff.cmake)
@@ -309,7 +309,7 @@ else (build_errors)
   message (STATUS "Install path: ${CMAKE_INSTALL_PREFIX}")
 
   if (BUILD_GAZEBO)
-    set(TEST_TYPE "UNIT")
+#    set(TEST_TYPE "UNIT")
     add_subdirectory(gazebo)
     add_subdirectory(media)
     add_subdirectory(test)
