From 343bcaed02f69e0c640c183098092099a2228ca9 Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hallen@kns.com>
Date: Thu, 20 Dec 2018 18:17:51 -0500
Subject: [PATCH] Remove python3 search for boost

---
 CMakeLists.txt | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 997bef3..85827d0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,11 +5,11 @@ find_package(catkin REQUIRED COMPONENTS rosconsole sensor_msgs)
 
 if(NOT ANDROID)
   find_package(PythonLibs)
-  if(PYTHONLIBS_VERSION_STRING VERSION_LESS 3)
-    find_package(Boost REQUIRED python)
-  else()
-    find_package(Boost REQUIRED python3)
-  endif()
+#  if(PYTHONLIBS_VERSION_STRING VERSION_LESS 3)
+# find_package(Boost REQUIRED python)
+#  else()
+#    find_package(Boost REQUIRED python3)
+#  endif()
 else()
 find_package(Boost REQUIRED)
 endif()
-- 
2.20.1

