diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0cebaf8..9ec6e4a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,11 +5,7 @@ find_package(catkin REQUIRED COMPONENTS rosconsole sensor_msgs)
 
 if(NOT ANDROID)
   find_package(PythonLibs)
-  if(PYTHONLIBS_VERSION_STRING VERSION_LESS 3)
-    find_package(Boost REQUIRED python)
-  else()
-    find_package(Boost REQUIRED python3)
-  endif()
+  find_package(Boost REQUIRED python)
 else()
 find_package(Boost REQUIRED)
 endif()
