From 610038af6aed8f20253be1374931b72399deb03b Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Wed, 14 Feb 2018 22:10:11 +0000
Subject: [PATCH] Exclude all test packages.

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 8d8c823..2400279 100644
--- a/setup.py
+++ b/setup.py
@@ -12,7 +12,7 @@ if sys.version_info < (3, 5):
 setup(
     name='ament_tools',
     version='0.4.0',
-    packages=find_packages(exclude=['test']),
+    packages=find_packages(exclude=['test', 'test.*']),
     install_requires=['ament-package', 'osrf_pycommon'],
     data_files=[
         ('share/ament_tools/environment',
-- 
2.16.1
