--- CMakeLists.txt	2019-03-05 14:45:35.000000000 +0900
+++ CMakeLists.txt.edit	2019-03-06 11:36:01.000000000 +0900
@@ -7,8 +7,21 @@
 if(PYTHONLIBS_VERSION_STRING VERSION_LESS 3)
-  find_package(Boost REQUIRED COMPONENTS filesystem python)
+  find_package(Boost REQUIRED COMPONENTS filesystem python27)
 else()
-  find_package(Boost REQUIRED COMPONENTS filesystem python3)
+	if(PYTHONLIBS_VERSION_STRING VERSION_GREATER 3.6)
+	  find_package(Boost REQUIRED COMPONENTS filesystem python36)
+	else()
+		if(PYTHONLIBS_VERSION_STRING VERSION_GREATER 3.5)
+		  find_package(Boost REQUIRED COMPONENTS filesystem python35)
+		else()
+			if(PYTHONLIBS_VERSION_STRING VERSION_GREATER 3.4)
+			  find_package(Boost REQUIRED COMPONENTS filesystem python34)
+			else()
+			  message(FATAL_ERROR, "At least Python 3.4 is needed for python 3, please consider migrating.")
+			endif()
+		endif()
+	endif()
 endif()
+
 include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
 
 catkin_python_setup()
