From 5ae34860e8983af7044c7e64b6ecc48f6e969f08 Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Wed, 24 Jan 2018 03:26:30 +0000
Subject: [PATCH] Fix Cmake

---
 CMakeLists.txt | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cac163f..8884891 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,9 +20,10 @@ target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
 
 
 
-catkin_add_gtest(utest_inds_utils test/utest.cpp)
-target_link_libraries(utest_inds_utils ${PROJECT_NAME} ${catkin_LIBRARIES})
-
+if(CATKIN_ENABLE_TESTING)
+  catkin_add_gtest(utest_inds_utils test/utest.cpp)
+  target_link_libraries(utest_inds_utils ${PROJECT_NAME} ${catkin_LIBRARIES})
+endif()
 
 
 install(
-- 
2.16.0
