--- a/CMakeLists.txt	2018-10-26 10:26:22.809392103 +0100
+++ b/CMakeLists.txt	2018-10-26 10:27:14.145391975 +0100
@@ -225,6 +225,8 @@
 # TODO(hrapp): This should not explicitly list pthread and use
 # PROTOBUF_LIBRARIES, but that failed on first try.
 target_link_libraries(${PROJECT_NAME} PUBLIC ${PROTOBUF_LIBRARY} pthread)
+target_link_libraries(${PROJECT_NAME} PUBLIC gmock)
+target_link_libraries(${PROJECT_NAME} PUBLIC gtest)
 if(${BUILD_GRPC})
   target_link_libraries(${PROJECT_NAME} PUBLIC grpc++)
   target_link_libraries(${PROJECT_NAME} PUBLIC async_grpc)
