diff --git a/KDLConfig.cmake.in b/KDLConfig.cmake.in
index 13a476b..40899c1 100644
--- a/KDLConfig.cmake.in
+++ b/KDLConfig.cmake.in
@@ -6,7 +6,7 @@
 
 # Compute paths
 get_filename_component(SELF_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
-set(orocos_kdl_INCLUDE_DIRS "@KDL_INCLUDE_DIRS@;@CMAKE_INSTALL_PREFIX@/include")
+set(orocos_kdl_INCLUDE_DIRS "@KDL_INCLUDE_DIRS@;$ENV{DEST_SETUP_DIR}/include")
 
 if(NOT TARGET orocos-kdl)
   include("${SELF_DIR}/OrocosKDLTargets.cmake")
@@ -15,4 +15,4 @@ endif()
 set(orocos_kdl_LIBRARIES orocos-kdl)
 
 # where the .pc pkgconfig files are installed
-set(orocos_kdl_PKGCONFIG_DIR "@CMAKE_INSTALL_PREFIX@/lib/pkgconfig")
+set(orocos_kdl_PKGCONFIG_DIR "$ENV{DEST_SETUP_DIR}/lib/pkgconfig")
