From db9c8fc962273f0ebb74ed7dd246c1ad715622ed Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Mon, 26 Feb 2018 15:51:53 -0500
Subject: [PATCH] Don't require argparse for installation if python is >= 2.7.

---
 setup.py | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e7d2524..a993264 100755
--- a/setup.py
+++ b/setup.py
@@ -1,14 +1,21 @@
 #!/usr/bin/env python
 
 import os
+import sys
 from setuptools import setup, find_packages
 
+install_requires=['catkin_pkg >= 0.1.28', 'rosdistro >= 0.5.0', 'rospkg', 'PyYAML', 'setuptools']
+
+# argparse is a part of the standard library since python 2.7
+if sys.version_info[0] == 2 and sys.version_info[1] <= 6:
+    install_requires.append('argparse')
+
 exec(open(os.path.join(os.path.dirname(__file__), 'src', 'rosinstall_generator', '__init__.py')).read())
 
 setup(
     name='rosinstall_generator',
     version=__version__,
-    install_requires=['argparse', 'catkin_pkg >= 0.1.28', 'rosdistro >= 0.5.0', 'rospkg', 'PyYAML', 'setuptools'],
+    install_requires=install_requires,
     packages=find_packages('src'),
     package_dir={'': 'src'},
     scripts=['bin/rosinstall_generator'],
