--- a/libstage/world.cc	2017-10-06 14:47:28.643577020 +0200
+++ b/libstage/world.cc	2017-10-06 14:47:59.390795105 +0200
@@ -803,8 +803,8 @@
   // Graphics Gems IV  
   const int32_t sx(sgn(dx));  
   const int32_t sy(sgn(dy));  
-  const int32_t ax(abs(dx)); 
-  const int32_t ay(abs(dy));  
+  const int32_t ax(std::abs(dx)); 
+  const int32_t ay(std::abs(dy));  
   const int32_t bx(2*ax);	
   const int32_t by(2*ay);	
   int32_t exy(ay-ax); // difference between x and y distances
@@ -1033,8 +1033,8 @@
       const int32_t dy( end.y - start.y );
       const int32_t sx(sgn(dx));  
       const int32_t sy(sgn(dy));  
-      const int32_t ax(abs(dx));  
-      const int32_t ay(abs(dy));  
+      const int32_t ax(std::abs(dx));  
+      const int32_t ay(std::abs(dy));  
       const int32_t bx(2*ax);	
       const int32_t by(2*ay);	 
 
