From 912af0ca0972967a76ade0bd679144aa8aa12219 Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Thu, 22 Mar 2018 21:45:53 +0000
Subject: [PATCH 2/2] Explicitly link gmock and gtest.

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c502769..da8519f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,6 +183,8 @@ target_include_directories(${PROJECT_NAME} SYSTEM PUBLIC
 # TODO(hrapp): This should not explicitly list pthread and use
 # PROTOBUF_LIBRARIES, but that failed on first try.
 target_link_libraries(${PROJECT_NAME} PUBLIC ${PROTOBUF_LIBRARY} pthread)
+target_link_libraries(${PROJECT_NAME} PUBLIC gmock)
+target_link_libraries(${PROJECT_NAME} PUBLIC gtest)
 
 # Add the binary directory first, so that port.h is included after it has
 # been generated.
-- 
2.16.2
