From 8c3b62f1a40fb40f1d74b6b8f703ef83c69dd88a Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Thu, 22 Feb 2018 21:56:52 +0000
Subject: [PATCH] Change lib to lib64.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 126aeb3..3b237d7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@ project(ros_workspace)
 
 find_package(ament_cmake_core REQUIRED)
 
-set(PYTHON_INSTALL_DIR "lib/python3.5/site-packages")
+set(PYTHON_INSTALL_DIR "lib64/python3.5/site-packages")
 set(ament_cmake_package_templates_ENVIRONMENT_HOOK_PYTHONPATH "${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_DIR}/ament_package/template/environment_hook/pythonpath.sh.in")
 ament_environment_hooks("${ament_cmake_package_templates_ENVIRONMENT_HOOK_PYTHONPATH}")
 
-- 
2.16.2
