diff --git a/cfg/TheoraPublisher.cfg b/cfg/TheoraPublisher.cfg
index 01b1224..5208ef7 100755
--- a/cfg/TheoraPublisher.cfg
+++ b/cfg/TheoraPublisher.cfg
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/ros-python-kinetic
 
 PACKAGE='theora_image_transport'
 
diff --git a/cfg/TheoraSubscriber.cfg b/cfg/TheoraSubscriber.cfg
index deec68b..11563ee 100755
--- a/cfg/TheoraSubscriber.cfg
+++ b/cfg/TheoraSubscriber.cfg
@@ -1,4 +1,4 @@
-#! /usr/bin/env python
+#! /usr/bin/ros-python-kinetic
 
 PACKAGE='theora_image_transport'
 
