diff --git a/cmake/catkin_generate_environment.cmake b/cmake/catkin_generate_environment.cmake
index 912078b..5f31710 100644
--- a/cmake/catkin_generate_environment.cmake
+++ b/cmake/catkin_generate_environment.cmake
@@ -72,7 +72,11 @@ function(catkin_generate_environment)
     @ONLY)
 
   # installspace
-  set(SETUP_DIR ${CMAKE_INSTALL_PREFIX})
+  if(DEFINED ENV{DEST_SETUP_DIR})
+    set(SETUP_DIR "$ENV{DEST_SETUP_DIR}")
+  else()
+    set(SETUP_DIR "$ENV{CMAKE_INSTALL_PREFIX}")
+  endif()
 
   if(NOT CATKIN_BUILD_BINARY_PACKAGE)
     # install empty workspace marker if it doesn't already exist
