diff --git a/cmake/all.cmake b/cmake/all.cmake
index 6910906..a7727b5 100644
--- a/cmake/all.cmake
+++ b/cmake/all.cmake
@@ -52,6 +52,10 @@ set(CMAKE_PREFIX_PATH_AS_IS ${CMAKE_PREFIX_PATH})
 
 # list of unique catkin workspaces based on CMAKE_PREFIX_PATH
 set(CATKIN_WORKSPACES "")
+if (NOT DEFINED CATKIN_PREFIX_PATH)
+  set(CATKIN_PREFIX_PATH ${CMAKE_PREFIX_PATH})
+endif()
+
 foreach(path ${CMAKE_PREFIX_PATH})
   if(EXISTS "${path}/.catkin")
     list(FIND CATKIN_WORKSPACES ${path} _index)
