From 55c980d52a07f6cdc84acd7b0c8692cbb20b1d07 Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hallen@kns.com>
Date: Sat, 8 Dec 2018 18:50:21 -0500
Subject: [PATCH 1/2] Fix CMake to not do tests

---
 CMakeLists.txt | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 15046db..84dc0bd 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,6 +1,6 @@
 # Catkin User Guide: http://www.ros.org/doc/groovy/api/catkin/html/user_guide/user_guide.html
 # Catkin CMake Standard: http://www.ros.org/doc/groovy/api/catkin/html/user_guide/standards.html
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.0.2)
 project(willow_maps)
 # Load catkin and all dependencies required for this package
 # TODO: remove all from COMPONENTS that are not catkin packages.
@@ -25,11 +25,11 @@ find_package(catkin REQUIRED )
 # CATKIN_MIGRATION: removed during catkin migration
 # rosbuild_init()
 
-catkin_download_test_data(download_data_willow-willow-full-0.025.pgm http://download.ros.org/data/maps/willow-full-0.025.pgm FILENAME willow-full-0.025.pgm MD5 e66b17ee374f2d7657972efcb3e2e4f7)
-catkin_download_test_data(download_data_willow-willow-full-0.05.pgm http://download.ros.org/data/maps/willow-full-0.05.pgm FILENAME willow-full-0.05.pgm  MD5 b61694296e08965096c5e78611fd9765)
-catkin_download_test_data(download_data_willow-willow-full-0.10.pgm http://download.ros.org/data/maps/willow-full-0.10.pgm FILENAME willow-full-0.10.pgm MD5 b84465cdbbfe3e2fb9eb4579e0bcaf0e)
-catkin_download_test_data(download_data_willow-willow-sans-whitelab-0.025.pgm http://download.ros.org/data/maps/willow-sans-whitelab-0.025.pgm FILENAME willow-sans-whitelab-0.025.pgm  MD5 dce0081e83a9e2eb8b29c51c8cd3ffac)
-catkin_download_test_data(download_data_willow-willow.tmap http://download.ros.org/data/maps/willow.tmap FILENAME willow.tmap MD5 48864073990c66173dade2bfcae5669d)
+# file(DOWNLOAD http://download.ros.org/data/maps/willow-full-0.025.pgm willow-full-0.025.pgm EXPECTED_MD5 e66b17ee374f2d7657972efcb3e2e4f7)
+# file(DOWNLOAD -0.05.pgm http://download.ros.org/data/maps/willow-full-0.05.pgm willow-full-0.05.pgm  EXPECTED_MD5 d41d8cd98f00b204e9800998ecf8427e)
+# file(DOWNLOAD -0.10.pgm http://download.ros.org/data/maps/willow-full-0.10.pgm willow-full-0.10.pgm EXPECTED_MD5 d41d8cd98f00b204e9800998ecf8427e)
+# file(DOWNLOAD -whitelab-0.025.pgm http://download.ros.org/data/maps/willow-sans-whitelab-0.025.pgm $(pwd)/willow-sans-whitelab-0.025.pgm EXPECTED_MD5 d41d8cd98f00b204e9800998ecf8427e)
+# file(DOWNLOAD http://download.ros.org/data/maps/willow.tmap $(pwd)/willow.tmap EXPECTED_MD5 48864073990c66173dade2bfcae5669d)
 #catkin_download_test_data(download_data_willow-willow-2010-02-18-0.025.pgm http://download.ros.org/data/maps/willow-2010-02-18-0.025.pgm FILENAME willow-2010-02-18-0.025.pgm  MD5 1f82c191f1e4a830669de7bbc4bc989a)
 #catkin_download_test_data(download_data_willow-willow-sans-whitelab-2010-02-18-0.025.pgm http://download.ros.org/data/maps/willow-sans-whitelab-2010-02-18-0.025.pgm FILENAME willow-sans-whitelab-2010-02-18-0.025.pgm MD5 9fc85f28c98034979f10e8f6ece676bb)## Generate added messages and services with any dependencies listed here
 
-- 
2.19.2

