diff --git a/test/test_scan_filter_chain.cpp b/test/test_scan_filter_chain.cpp
index b661f5f..93548fa 100644
--- a/test/test_scan_filter_chain.cpp
+++ b/test/test_scan_filter_chain.cpp
@@ -60,8 +60,8 @@ sensor_msgs::LaserScan gen_msg(){
  */
 void expect_ranges_eq(const std::vector<float> &a, const std::vector<float> &b) {
   for( int i=0; i<10; i++) {
-    if(isnan(a[i])) {
-      EXPECT_TRUE(isnan(a[i]));
+    if(std::isnan(a[i])) {
+      EXPECT_TRUE(std::isnan(b[i]));
     }
     else {
       EXPECT_NEAR(a[i], b[i], 1e-6);
