diff --git a/CMakeLists.txt b/CMakeLists.txt
index 33c48f7..c7b91c4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -186,10 +186,12 @@ set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}${WARNING_CXX_FLAGS} ${UNFILTERED_FLAGS}
 # OS Specific initialization
 if (UNIX)
   sdf_setup_unix()
-else (WIN32)
+elseif (WIN32)
   sdf_setup_windows()
-else (APPLE)
+elseif (APPLE)
   sdf_setup_apple()
+else ()
+  build_error("unknown platform")
 endif()
 
 #################################################
