--- cmake/python.cmake	2017-03-07 00:58:50.000000000 +0900
+++ cmake/python.cmake.fix	2019-03-08 13:40:45.000000000 +0900
@@ -11,9 +11,9 @@
 set(_PYTHON_PATH_VERSION_SUFFIX "${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}")
 
 set(enable_setuptools_deb_layout OFF)
-if(EXISTS "/etc/debian_version")
-  set(enable_setuptools_deb_layout ON)
-endif()
+#if(EXISTS "/etc/debian_version")
+#  set(enable_setuptools_deb_layout ON)
+#endif()
 option(SETUPTOOLS_DEB_LAYOUT "Enable debian style python package layout" ${enable_setuptools_deb_layout})
 
 if(SETUPTOOLS_DEB_LAYOUT)
