From c054f32d924d965685885d36b1b0e7c63ce9115c Mon Sep 17 00:00:00 2001
From: "Hunter L. Allen" <hunter@openrobotics.org>
Date: Wed, 24 Jan 2018 01:55:51 +0000
Subject: [PATCH] Fix cmake file

---
 CMakeLists.txt | 35 ++++++++++++++++++++---------------
 1 file changed, 20 insertions(+), 15 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 06830d5..b4198ef 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -45,7 +45,7 @@ set(SRC_FILES src/byte_array.cpp
 	src/messages/robot_status_message.cpp
 
 	src/simple_comms_fault_handler.cpp)
-					
+
 set(UTEST_SRC_FILES test/utest.cpp test/utest_message.cpp)
 
 # The simple message make file builds two libraries: simple_message and
@@ -101,9 +101,11 @@ add_library(simple_message ${SRC_FILES})
 target_link_libraries(simple_message ${catkin_LIBRARIES})
 add_dependencies(simple_message ${industrial_msgs_EXPORTED_TARGETS})
 
-catkin_add_gtest(utest ${UTEST_SRC_FILES})
-set_target_properties(utest PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=11000")
-target_link_libraries(utest simple_message)
+if(CATKIN_ENABLE_TESTING)
+  catkin_add_gtest(utest ${UTEST_SRC_FILES})
+  set_target_properties(utest PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=11000")
+  target_link_libraries(utest simple_message)
+endif()
 
 # ALTERNATIVE LIBRARY (DIFFERENT ENDIAN)
 add_library(simple_message_bswap ${SRC_FILES})
@@ -111,9 +113,11 @@ set_target_properties(simple_message_bswap PROPERTIES COMPILE_DEFINITIONS "BYTE_
 target_link_libraries(simple_message_bswap ${catkin_LIBRARIES})
 add_dependencies(simple_message_bswap ${industrial_msgs_EXPORTED_TARGETS})
 
-catkin_add_gtest(utest_byte_swapping ${UTEST_SRC_FILES})
-set_target_properties(utest_byte_swapping PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=12000")
-target_link_libraries(utest_byte_swapping simple_message_bswap)
+if(CATKIN_ENABLE_TESTING)
+  catkin_add_gtest(utest_byte_swapping ${UTEST_SRC_FILES})
+  set_target_properties(utest_byte_swapping PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=12000")
+  target_link_libraries(utest_byte_swapping simple_message_bswap)
+endif()
 
 # ALTERNATIVE LIBRARY (64-bit floats)
 add_library(simple_message_float64 ${SRC_FILES})
@@ -121,19 +125,20 @@ set_target_properties(simple_message_float64 PROPERTIES COMPILE_DEFINITIONS "FLO
 target_link_libraries(simple_message_float64 ${catkin_LIBRARIES})
 add_dependencies(simple_message_float64 ${industrial_msgs_EXPORTED_TARGETS})
 
-catkin_add_gtest(utest_float64 ${UTEST_SRC_FILES})
-set_target_properties(utest_float64 PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=13000;FLOAT64")
-target_link_libraries(utest_float64 simple_message_float64)
+if(CATKIN_ENABLE_TESTING)
+  catkin_add_gtest(utest_float64 ${UTEST_SRC_FILES})
+  set_target_properties(utest_float64 PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=13000;FLOAT64")
+  target_link_libraries(utest_float64 simple_message_float64)
 
-catkin_add_gtest(utest_udp ${UTEST_SRC_FILES})
-set_target_properties(utest_udp PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=15000;UDP_TEST")
-target_link_libraries(utest_udp simple_message)
+  catkin_add_gtest(utest_udp ${UTEST_SRC_FILES})
+  set_target_properties(utest_udp PROPERTIES COMPILE_DEFINITIONS "TEST_PORT_BASE=15000;UDP_TEST")
+  target_link_libraries(utest_udp simple_message)
+endif()
 
 install(
-    TARGETS simple_message simple_message_bswap simple_message_float64 
+    TARGETS simple_message simple_message_bswap simple_message_float64
     LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION})
 
 install(
     DIRECTORY include/${PROJECT_NAME}/
     DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION})
-
-- 
2.16.0
