From b1d85850ceab9e1ddb2b255574d9e6e299717622 Mon Sep 17 00:00:00 2001
From: Sammy Pfeiffer <sammypfeiffer@gmail.com>
Date: Sat, 28 Apr 2018 20:42:01 +1000
Subject: [PATCH] If testing is not enabled, it should not run

---
 CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 553cf50..0bc0c51 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -177,5 +177,7 @@ catkin_package(
 #   target_link_libraries(${PROJECT_NAME}-test ${PROJECT_NAME})
 # endif()
 
-## Add folders to be run by python nosetests
-catkin_add_nosetests(test)
+if(CATKIN_ENABLE_TESTING)
+  ## Add folders to be run by python nosetests
+  catkin_add_nosetests(test)
+endif()
\ No newline at end of file
-- 
1.9.1

